### PR TITLE
Support multiple owners

### PR DESCRIPTION
**Pull request by opoku**

### CODE REVIEW

**Review by semperos — CHANGES_REQUESTED** [review]

File: pivotal-import/pivotal_import.py [hunk 1]
```diff
@@ -13,8 +13,11 @@
 parser = argparse.ArgumentParser(
     description="Imports the Pivotal Tracker CSV export to Shortcut",
 )
-parser.add_argument("--apply", action="store_true", required=False)
-parser.add_argument("--bulk", action="store_true", required=False)
+parser.add_argument(
+    "--apply", action="store_true", help="Actually creates the entities inside Shortcut"
+)
+parser.add_argument("--debug", action="store_true", help="Turns on debugging logs")
+
```
[the remark this answers is not in the document]
Comment (opoku) [reply pivotal-import/pivotal_import.py:20]: bloop. moving too fast

[the remark this answers is not in the document]
Comment (opoku) [reply pivotal-import/pivotal_import.py:20]: I decided to kill that option all together. We should always just be using the bulk story create option

[the remark this answers is not in the document]
Comment (opoku) [reply pivotal-import/pivotal_import.py:20]: Should be fixed now.

**Review by semperos — APPROVED** [review]
Looks good and multiple owners works 👍🏻

**Pull request merged**